packages/webapp/public/locales/fr/translation.json

```diff
@@ -1,14 +1,14 @@
 {
   "ADD_FARM": {
     "ADDRESS_IS_REQUIRED": "Address civique est requise",
-    "DISABLE_GEO_LOCATION": "",
+    "DISABLE_GEO_LOCATION": "Les services de localisation doivent être activés pour trouver votre position actuelle",
```

Comment:
There are three empty string values in this file.
(Nothing to do with this line; I can only put review comments on lines that were changed in the last update.)